src/main/java/dev/slimevr/vr/processor/skeleton/LegTweakBuffer.java

```diff
@@ -363,15 +403,11 @@ public void calculateFootAttributes(boolean active) {
 	public void updateFrameNumber(int frameNumber) {
 		this.frameNumber = frameNumber;
 
-		if (this.frameNumber >= 10) {
-			this.parent = null; // once a frame is 10 frames old, it is no
-								// longer
-								// needed
-		}
+		if (this.frameNumber >= 10)
+			this.parent = null;
```

Comment:
Why remove this comment? 🤔

Reply:
It was out of place right there I added back what it was saying in the functions header comment